apps/renterd/components/Files/FilesCmd/FilesSearchCmd/index.tsx

```diff
@@ -30,15 +32,21 @@ export function FilesSearchCmd({
   const { activeBucketName: activeBucket, navigateToModeSpecificFiltering } =
     useFilesManager()
   const onSearchPage = currentPage?.namespace === filesSearchPage.namespace
-  const searchBucket = activeBucket || 'default'
-  const results = useObjects({
-    disabled: !onSearchPage,
-    params: {
-      bucket: searchBucket,
+  const params = useMemo(() => {
+    const p: ObjectsParams = {
       prefix: debouncedSearch,
       limit: 10,
       delimiter: '',
-    },
+    }
+    if (activeBucket) {
+      p.bucket = activeBucket
```

Comment:
Because the bucket is optional, search results might contain objects from various buckets, every object however is still tied to a bucket. I feel this PR is missing a change to whatever UI component is displaying the results to show that extra piece of context, which you can now get from the object's metadata by the way. (`bucket` column somewhere essentially)

Reply:
Are we ever without an active bucket in the current UI? If not I'd treat that as a separate feature for when we add a view that allows for searching without an active bucket.

Reply:
If you activated file search from the cmd+k menu it would search globally. I've update this PR to make that UI more clear and added distinct search all vs search in bucket options to the cmd+k menu. I'll share a video with you both.

Reply:
🔥